build/snap/snapcraft.yaml

```diff
@@ -0,0 +1,56 @@
+name: eksctl
+summary: CLI tool.
+description: a CLI for Amazon EKS
+  
+adopt-info: eksctl
+
+grade: stable
+confinement: strict
+base: core18
+
+architectures:
+  - build-on: i386
+  - build-on: amd64
+  - build-on: armhf
+  - build-on: arm64
+
+apps:
+  eksctl:
+    command: bin/eksctl
+    plugs:
+      - home
+      - network
+      - docker
+      - removable-media
+
+parts:
+  eksctl:
+    plugin: nil
+    source: https://github.com/weaveworks/eksctl.git
+    source-type: git
+    override-pull: |
+      git clone https://github.com/weaveworks/eksctl.git src/github.com/weaveworks/eksctl
```

Comment:
Does snapcraft run builds for us, or do we have an option to upload? I'd expect with goreleaser we'd be uploading the same binaries we upload to GitHub, which is preferred.